rockstar/rockstar.js

```diff
@@ -51,6 +75,15 @@
         }, 200);        
         exportObjects();
         //createPrefixA("<li>", "Export Objects", "#nav-admin-reports-2", exportObjects);
+
+        if (location.pathname == "/admin/users") {

```

Comment:
there's already an if statement for this above

Reply:
Maybe I misunderstood you, but I moved this so that the "Deleted XYZ" menu item shows below all the "Export ABC" menu items, but above the "API Explorer" menu item.